inc/saineighbor.h

```diff
@@ -115,6 +115,17 @@ typedef enum _sai_neighbor_entry_attr_t
      */
     SAI_NEIGHBOR_ENTRY_ATTR_COUNTER_ID,
 
+    /**
```

Comment:
For the SAI_NEIGHBOR_ENTRY_ATTR_DST_MAC_ADDRESS ATTR, I believe we discussed that we could pass in a DUMMY or a NULL mac address. Could we add a comment to that effect?

Reply:
Will add comment

Reply:
Added comment "validonly SAI_NEIGHBOR_ENTRY_ATTR_IS_LOCAL == true".  The "@validonly" meta field fails the metachecker since SAI_NEIGHBOR_ENTRY_ATTR_COUNTER_ID is mandatory on create.